spec/glueby/wallet_spec.rb

```diff
@@ -0,0 +1,25 @@
+RSpec.describe 'Glueby::Wallet' do
+  class TestWalletAdapter < Glueby::Wallet::AbstractWalletAdapter
+    def create_wallet; end
+  end
+
+  before do
+    Glueby::Wallet.wallet_adapter = TestWalletAdapter.new
+  end
+
```

Comment:
No, it is for now. 

`Glueby::Wallet.wallet_adapter` is used only in that `wallet_spec.rb`.

Reply:
For example, it supposes that Glueby::Contract::Hoge implements new methods `balance`.

```ruby
  def balance
    @wallet ||= Glueby::Internal::Wallet.create
    @wallet.balance
  end
```

and then, write spec:

```ruby
  describe 'balance' do
    context 'no adapter' do
      it 'should raise error' do
        expect{ balance }.to raise_error(Glueby::Internal::Wallet::Errors::ShouldInitializeWalletAdapter)
      end
    end
  end
```

It should raise ShouldInitializeWalletAdapter error and this specs will pass when we test this spec only.
But when we test this spec after executing the spec "wallet_spec.rb:11", it will fail.

```
bundle exec rspec ./spec/glueby/internal/wallet_spec.rb:11 ./spec/glueby/contract/hoge.rb
```

To avoid unexpected fails in new specs in the future, it is preferred to add `after` statement and keep state of system be equal before and after the test.

Reply:
ここで `after` ブロックが必要だという話は、

- あるテストケースで作ったデータはそのテストケースで削除しておきましょう
- あるテストケースでロックしたファイルはそのテストケースでクローズしておきましょう

と同じ話で、テストケース内で作ったままのadapterをそのままにしておくと、別のテストケースで失敗する可能性があり、その場合に原因が特定しにくいと思っています。

Reply:
仰るとおりだと思います。修正します。

Reply:
Fixed at a9236337f9b7b29c7dfb147524de2a13691d47a5